test-tools/core/src/main/java/org/kie/cloud/tests/core/resources/Project.java

```diff
@@ -0,0 +1,335 @@
+            oc.execute(args.toArray(new String[0]));
+        } catch (InterruptedException e) {
+            Thread.currentThread().interrupt();
+            throw new RuntimeException("Interrupted while waiting for scenario to be initialized.", e);
+        } finally {
+            semaphore.release();
+        }
+    }
+
+    private String getApbCommand(List<String> args) {
+        return args.stream().collect(Collectors.joining(" "));
+    }
+
+    private String formatExtraVars(Map<String, String> extraVars) {
+        return extraVars.entrySet()
+                .stream()
+                .map(entry -> "\"" + entry.getKey() + "\":\"" + entry.getValue() + "\"")
+                .collect(Collectors.joining(", ", "{", "}"));
+    }
+
+    /**
+     * Create all resources defined in resource URL.
+     *
+     * @param resourceUrl URL of resource list to be created
+     */
+    public void createResources(String resourceUrl) {
+        try {
+            KubernetesList resourceList = openShift.lists().inNamespace(projectName).load(new URL(resourceUrl)).get();
+            openShift.lists().inNamespace(projectName).create(resourceList);
+        } catch (MalformedURLException e) {
+            throw new RuntimeException("Malformed resource URL", e);
+        }
+    }
+
+    /**
+     * Create resources from YAML file using command line client.
+     * @param yamlUrl Url to yaml file with resources
+     */
+    public void createResourcesFromYaml(String yamlUrl) {
+        final String output = openShiftBinaryClient().execute("create", "-f", yamlUrl);
+        LOGGER.info("Yaml resources from file {} were created by oc client. Output = {}", yamlUrl, output);
+    }
+
+    /**
+     * Create resources from YAML files using command line client.
+     * @param yamlUrls Urls to yaml files with resources
+     */
+    public void createResourcesFromYaml(List<String> yamlUrls) {
+        final OpenShiftBinary oc = openShiftBinaryClient();
+        for (String url : yamlUrls) {
+            final String output = oc.execute("create", "-f", url);
+            LOGGER.info("Yaml resources from file {} were created by oc client. Output = {}", url, output);
+        }
+    }
+
+    /**
+     * Create resources from YAML files as admin using command line client.
+     * @param yamlUrl Url to yaml files with resources
+     */
+    public void createResourcesFromYamlAsAdmin(String yamlUrl) {
+        final String output = openShiftBinaryAdminClient().execute("create", "-f", yamlUrl);
+        LOGGER.info("Yaml resources from file {} were created by oc client. Output = {}", yamlUrl, output);
+    }
+
+    /**
+     * Create resources from YAML files as admin using command line client.
+     * @param yamlUrls Url to yaml files with resources
+     */
+    public void createResourcesFromYamlAsAdmin(List<String> yamlUrls) {
+        final OpenShiftBinary oc = openShiftBinaryAdminClient();
+        for (String url : yamlUrls) {
```

Comment:
Same here as for createResourcesFromYaml

Reply:
done